pkg/framework/profile/profile.go

```diff
@@ -306,6 +306,7 @@ func (d profileImpl) RunDeschedulePlugins(ctx context.Context, nodes []*v1.Node)
 		ctx, span = tracing.Tracer().Start(ctx, pl.Name(), trace.WithAttributes(attribute.String("plugin", pl.Name()), attribute.String("profile", d.profileName), attribute.String("operation", tracing.DescheduleOperation)))
 		defer span.End()
 		evicted := d.podEvictor.TotalEvicted()
+		evictionRequests := d.podEvictor.TotalEvictionRequests()
```

Comment:
It would be easier to track if we prefixed these, for example `evictionRequestsBeforeBalance`?

Reply:
Done